apps/mobile/src/app/components/Auction/AuctionTemplate.tsx

```diff
@@ -65,41 +68,104 @@ export function AuctionTemplate(props: Props) {
     refetch();
   };
 
-  return (
-    <ScrollView>
-      {isBidModal && <BidModal bid={bid} closeModal={closeModal} confirmModal={confirmModal} />}
-      <AuctionHeader />
-      <AuctionImage imageType={imageTypes.detail} />
-      <View my={5} mx={2} px={3} paddingTop={insets.top} paddingBottom={insets.bottom}>
-        {isLoading && <Loader />}
-        {!isLoading && !auction && <Text>Error while loading auction</Text>}
-        {!isLoading && !isError && auction && (
-          <>
-            <Text fontSize='lg' my={3}>
-              {auction.title}
-            </Text>
-            <Text fontSize='sm' my={3}>
-              {auction.description ?? ''}
-            </Text>
-          </>
-        )}
-        <AuctionAuthor
-          authorInitials={auction.author.initials}
-          avatarSrc={auction.author.avatarSrc}
-          authorFirstName={auction.author.firstName}
-          authorLastName={auction.author.lastName}
-          avatarBgColor={theme.colors.primary[500]}
+  if (!auction) {
+    return <Loader />;
+  }
+  const BidPanel = (
```

Comment:
I think each component should havehis own separate scope. Please create a separate BidPanel component in a separate scope